mne_bids/path.py

```diff
@@ -1029,6 +1029,104 @@ def _check_non_sub_ses_entity(bids_path):
     return False
 
 
+def _print_lines_with_entry(file, entry, folder, is_tsv, line_numbers,
+                            outfile):
+    """Print the lines that contain the entry."""
```

Comment:
I know it's a private function, but since there's quite a few arguments, is it possible to add a doctoring here describing the different arguments and their purpose? That way future dev might be able to re-use this function w/o too much effort.